content/10.methods.md

```diff
@@ -30,7 +30,7 @@ We split names into first and last names as described for ISCB Fellows.
 #### RECOMB Keynote Speakers
 
 We examined the webpage for each RECOMB meeting.
-We were able to successfully find pages with keynote speakers for 1999, 2000, 2001, 2004, 2007, 2008, and 2010-2019.
+We were able to successfully find pages with keynote speakers for the years 1997-2019.
```

Comment:
I kept the one with gaps because the webpages do not have the additional years. The following sentence clarifies where the additional information came from.
```suggestion
We were able to successfully find pages with keynote speakers for 1999, 2000, 2001, 2004, 2007, 2008, and 2010-2019.

Reply:
Ah sorry, I missed the following sentence! I was editing yesterday's version. Should we clarify these as "webpages" or "conference webpages"?

Reply:
Also, I realized the 2016 proceedings have 2 tables with all speaker names from 1997-2016. Maybe we don't need to cite the other ones.

Reply:
I went with what was in the final column of the spreadsheet. You did the curation so I trust your judgement. I would reference what you used, just in case there are differences. I also think "conference webpages" is better than webpages, so 👍 on that change.

Reply:
When I found the second table in the 2016 proceedings, I went through all the years and double-checked all the names from 1997-2016! So there were not be any differences. I will comment on the spreadsheet for a couple of instances where talks were canceled due to emergency reasons but I still think we should include those names.